features/telemetry/protocols/bmp/parser/src/main/java/org/opennms/netmgt/telemetry/protocols/bmp/parser/proto/bmp/Packet.java

```diff
@@ -67,6 +69,7 @@ public void visit(PeerUpPacket packet) {}
             public void visit(InitiationPacket packet) {}
             public void visit(TerminationPacket packet) {}
             public void visit(RouteMirroringPacket packet) {}
+            public void visit(final UnknownPacket packet) {}
```

Comment:
Only one final? Or should we make all final?

Reply:
Yup.